src/groovy/org/pih/warehouse/api/StockMovement.groovy

```diff
@@ -209,8 +209,8 @@ class StockMovement {
 
     static StockMovement createFromShipment(Shipment shipment) {
 
-        String statusCode = (shipment.status.code == ShipmentStatusCode.SHIPPED) ?
-                RequisitionStatus.ISSUED.toString() : RequisitionStatus.PENDING.toString()
+        String statusCode = (shipment.status.code == ShipmentStatusCode.PENDING) ?
```

Comment:
@DominikaHatala Can you explain why this was necessary?

Reply:
@jmiranda Those changes are fixing ticket 3161. I will remove it from here and create another PR for this one. It fixes the issue with redirecting PO based stock movement when status code is PARTIALLY_RECEIVED or RECEIVED.